Gemfile.lock

```diff
@@ -309,7 +309,7 @@ GEM
     dotenv (0.10.0)
     dotenv-rails (0.10.0)
       dotenv (= 0.10.0)
-    dough-ruby (5.12.0.267)
+    dough-ruby (5.12.0.269)
```

Comment:
Hey @Tr4pSt3R, this is being handled in my PR - https://github.com/moneyadviceservice/frontend/pull/1430